ntbs-service/Migrations/20191021164821_MovePHECAdGroupToPHECTable.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations

```

Comment:
I tried adding a new join table first but ran into EF migration issues with the seed data (and incredibly unhelpful error messages) - I was stuck on this for about 2 hours, and felt it was best to move forward and just add PHECCode as a column to TBservice for now. Shall I raise a tech debt ticket to fix this?

Reply:
Ok let's just keep it where it is for now :)